pynest/nest/tests/test_multiple_synapses.py

```diff
@@ -292,6 +292,51 @@ def test_MultipleSynapses_make_symmetric(self):
 
         self.assertEqual(ref_conn_list, sorted_conn_list)
 
+    def test_MultipleSynapses_receptor_type(self):
+        """Test co-location of synapses with different receptor types"""
+        num_src = 7
+        num_trg = 7
+
+        src = nest.Create('iaf_psc_exp_multisynapse', num_src)
+        trgt = nest.Create('iaf_psc_exp_multisynapse', num_trg, {'tau_syn': [0.1 + i for i in range(num_trg)]})
+        node = nest.Create('iaf_psc_alpha')
+
+        syn_spec = nest.CollocatedSynapses({'synapse_model': 'stdp_synapse',
+                                            'weight': 5.,
+                                            'receptor_type': 2},
+                                           {'weight': 1.5, 'receptor_type': 7},
+                                           {'synapse_model': 'stdp_synapse', 'weight': 3, 'receptor_type': 5})
+
+        nest.Connect(src, trgt, 'one_to_one', syn_spec=syn_spec)
+        nest.Connect(node, node)  # should have receptor 0
```

Comment:
What is described in this comment is not checked in an assert.

Reply:
Done!